src/Features/Admin/Common Requests/mutation.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+import { baseUrl } from '../../../scripts/constants';
+
+export const editUserById = async ({ id, token, userData }) => {
+  const res = await axios.patch(baseUrl + 'users/' + id, userData, {
+    headers: {
+      Authorization: 'Bearer ' + token,
+    },
+  });
+  return res;
+};
+
+export const createUser = async ({ token, userData }) => {
+  const res = await axios.post(baseUrl + 'users', userData, {
+    headers: {
```

Comment:
Can't we declare the header function like above and just pass the token in it and call it several times that way we can reduced to code ..